### PR TITLE
Update Installation Guide

### PR DESCRIPTION
**Pull request by SerhiiOryshych**

### CODE REVIEW

File: src/get-started/install.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ with its compilation in certain configurations. If you have encountered a proble
 with installing Exonum on Windows, you may ask around on [Gitter](https://gitter.im/exonum/exonum)
 or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
+## Adding environment variables
+
+This variables needed for linking with the `rocksdb` and `snappy` libraries.
```
Comment (slowli) [inline src/get-started/install.md]: Language: "variables *are* needed"? Plus, they are not really required, at least from my experience; I didn't set these variables (more than that, one of my envs in Windows 8.1), but Exonum works just fine :thinking:

File: src/get-started/install.md [hunk 2]
```diff
@@ -66,6 +66,24 @@ with its compilation in certain configurations. If you have encountered a proble
...
+
+### macOS
```
Comment (slowli) [inline src/get-started/install.md]: Nitpick: Inconsistent capitalization ("macOS" vs "MacOS" in the "Dependencies" section).

File: src/get-started/install.md (hunk 1 above)
Comment (vitvakatu) [reply src/get-started/install.md]: Setting these vars improves compile times, afaik. Btw, @SerhiiOryshych, `these`, not `this`

File: src/get-started/install.md [hunk 3]
```diff
@@ -68,9 +64,9 @@ or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
 ## Adding environment variables
 
-This variables needed for linking with the `rocksdb` and `snappy` libraries.
+These variables are needed for linking with the `rocksdb` and `snappy` libraries.
```
Comment (aleksuss) [inline src/get-started/install.md]: If your OS contains pre-compiled libraries `rocksdb` or `snappy`, you may setup environment variables. This significantly reduce a compile time.

Comment (slowli) [reply src/get-started/install.md]: @SerhiiOryshych Could you please add this explanation to the document?

File: src/get-started/install.md [hunk 4]
```diff
@@ -66,6 +62,27 @@ with its compilation in certain configurations. If you have encountered a proble
 with installing Exonum on Windows, you may ask around on [Gitter](https://gitter.im/exonum/exonum)
 or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
+## Adding environment variables
+
+If your OS contains pre-compiled `rocksdb` or `snappy` libraries,
+you may setup `ROCKSDB_LIB_DIR` and/or `SNAPPY_LIB_DIR` environment variable
+to point to a directory with these libraries.
+This significantly reduce a compile time.
```
Comment (slowli) [inline src/get-started/install.md]: Grammar: "This will significantly reduce compile time."

**Pull request merged**